docs/CLI/installation.mdx

```diff
@@ -56,7 +56,13 @@ If you want to install ORAS on an Intel-based Mac, you can download it from `htt
 
 ### Windows
 
-Add `%USERPROFILE%\bin\` to your `PATH` environment variable so that `oras.exe` can be found.
+- You can install ORAS CLI on Windows using [WinGet (Windows Package Manager)](https://github.com/microsoft/winget-pkgs):
+
+```bash
+winget install oras
+```
+
+- Alternatively, you can install ORAS CLI using `.exe` installer. Add `%USERPROFILE%\bin\` to your `PATH` environment variable so that `oras.exe` can be found.
```

Comment:
```suggestion
- Alternatively, you can install ORAS CLI using `.exe` installer. Add `%USERPROFILE%\bin` to your `PATH` environment variable so that `oras.exe` can be found.
```